frontend/src/metabase/hooks/use-modal-open.ts

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from "react";
+
+export function useModalOpen() {
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setOpen(true);
+    });
+  }, []);
```

Comment:
is this so we get a transition? if so we should add a comment to that effect.

Reply:
Correct